src/promptflow-devkit/promptflow/_sdk/entities/_run.py

```diff
@@ -169,6 +169,19 @@ def __init__(
         if self._use_remote_flow:
             self._flow_name = parse_remote_flow_pattern(flow)
             self._lineage_id = self._flow_name
```

Comment:
update changelog

Reply:
Update after announce of flex flow & prompty?

Reply:
I thought we already announced promty, but turns out not..